app/views/shared/_filter_group.html.haml

```diff
@@ -1,76 +1,63 @@
 -# Usage to include filter group
 -#
--# = render partial: 'shared/filter_group', locals: {
+-# = render 'shared/filter_group', :f => <FormBuilder form>,
 -#   filters: {
--#     :title => 'filter group heading',
+-#     :title => <String>,
+-#     :removeButtons => <Boolean>,
+-#     :totalCount => <Integer>,
 -#     :items => [
--#       { :f => f, :options => <Array>, :title => 'Working patterns'},
--#       { :f => f, :options => <Array>, :title => 'Subjects', :search => true }
+-#       { :options => <Array>, :title => <String>, :search => <Boolean>, :key => <String> }
 -#     ]
 -#   }
--# }
 
-.accordion-group.govuk-accordion{ id: 'accordion-default', data: { module: "govuk-accordion"} }
-
-  - if filters.present? && filters[:control]
-    .govuk-accordion__section.govuk-accordion__title.moj-filter
-      .govuk-accordion__section-header.moj-filter__header
-        %h3.govuk-accordion__section-heading.accordion-group-heading
-          %span.govuk-accordion__section-button.moj-filter__heading-title{ id: "accordion-default-heading-title" }
-            %h2.govuk-heading-s
-              = filters[:title]
-
-      .govuk-accordion__section-content{ id: "accordion-default-content-title", aria: { labelledby: "accordion-default-heading-title" } }
-        .moj-filter__content
-          .moj-filter__selected
+- if filters.present?
+  .moj-filter
+    .moj-filter__header
+      .selected-filters__heading
+        %span.moj-filter__heading-title
+          %h2.govuk-heading-s
+            = filters[:title]
+        %a.govuk-link.govuk-link--no-visited-state{ href: '', id: 'close-all-groups' }
+          = 'Close all'
+
+    - if filters[:totalCount] > 0 && filters[:removeButtons]
+      .moj-filter__content
+        .moj-filter__selected
+          .selected-filters__heading{ class: 'govuk-!-margin-bottom-2' }
+            .govuk-body{ class: 'govuk-!-margin-bottom-0' }
+              = 'Selected filters'
+            %a.govuk-link.govuk-link--no-visited-state{ href: '', id: 'clear-filters-button' }
+              = 'Clear filters'
+
+          - filters[:items].each do |group|
             .govuk-heading-s{ class: 'govuk-!-margin-bottom-0' }
-              = 'Subjects'
-
-            .moj-filter__heading-action
-              %p
-                %a.govuk-link.govuk-link--no-visited-state{ href: ""}
-                  = 'Clear filters'
+              = group[:title]
 
             %ul.moj-filter-tags
-              %li
-                %a.moj-filter__tag{ href: "/path/to/remove/this" }
-                  %span.govuk-visually-hidden
-                    = 'Remove this filter'
-                  = 'Art and design'
-                  %i.fa.fa-close
+              - group[:options].each do |school|
```

Comment:
If this is to be reused, I would change the variable name `school` to something that isn't so specific